journal-letters/original/07-chapter-7.adoc

```diff
@@ -1165,8 +1165,8 @@ My heart, I must needs say, was greatly affected with her exercise and concern,
 and was comforted in the sense of that living presence, that was with us at that season.
 When we had spent some time together, and were refreshed, and I to go away,
 she did most solemnly take her leave of me in great affection,
-and signified how kind she took my visit; withal seriously saying, Farewell dear John,
-if I never see thee more.
+and signified how kind she took my visit; withal seriously saying, '`Farewell dear John,
```

Comment:
Let's also switch this to double-quotes instead of single.

Reply:
thanks for your help...